chainservice/chainservice.go

```diff
@@ -75,6 +99,7 @@ func (cs *ChainService) HandleAction(ctx context.Context, actPb *iotextypes.Acti
 	if err != nil {
 		return err
 	}
+	chainIDmetrics(act)
```

Comment:
move to L108, so duplicate action won't be counted

Reply:
done